src/schemaText/bridge-text-en.json

```diff
@@ -1052,5 +1052,123 @@
+      "label": "Memory Overload Protection"
+    },
+    "buffer_mode": {
+      "desc": "Message buffer mode.\n<code>memory</code>: Buffer all messages in memory. The messages will be lost in case of EMQX node restart\n<code>disk</code>: Buffer all messages on disk. The messages on disk are able to survive EMQX node restart.\n<code>hybrid</code>: Buffer message in memory first, when up to certain limit (see <code>segment_bytes</code> config for more information), then start offloading messages to disk, Like <code>memory</code> mode, the messages will be lost in case of EMQX node restart.",
+      "label": "Buffer Mode"
+    },
+    "buffer_per_partition_limit": {
+      "desc": "Number of bytes allowed to buffer for each Pulsar partition. When this limit is exceeded, old messages will be dropped in a trade for credits for new messages to be buffered.",
+      "label": "Per-partition Buffer Limit"
+    },
+    "buffer_segment_bytes": {
+      "desc": "Applicable when buffer mode is set to <code>disk</code> or <code>hybrid</code>.\nThis value is to specify the size of each on-disk buffer file.",
+      "label": "Segment File Bytes"
+    },
+    "config_enable": {
+      "desc": "Enable (true) or disable (false) this Pulsar bridge.",
+      "label": "Enable or Disable"
+    },
+    "desc_name": {
+      "desc": "Bridge name, used as a human-readable description of the bridge.",
+      "label": "Bridge Name"
+    },
+    "desc_type": {
+      "desc": "The Bridge Type",
+      "label": "Bridge Type"
+    },
+    "batch_size": {
+      "desc": "Maximum number of individual requests to batch in a Pulsar message.",
+      "label": "Batch size"
+    },
+    "buffer": {
+      "desc": "Configure producer message buffer.\n\nTell Pulsar producer how to buffer messages when EMQX has more messages to send than Pulsar can keep up, or when Pulsar is down.",
+      "label": "Message Buffer"
+    },
+    "compression": {
+      "desc": "Compression method.",
+      "label": "Compression"
+    },
+    "message_key": {
+      "desc": "Template to render Pulsar message key.",
+      "label": "Message Key"
+    },
+    "local_topic": {
+      "desc": "MQTT topic or topic filter as data source (bridge input). If rule action is used as data source, this config should be left empty, otherwise messages will be duplicated in Pulsar.",
+      "label": "Source MQTT Topic"
+    },
+    "max_batch_bytes": {
+      "desc": "Maximum bytes to collect in a Pulsar message batch. Most of the Pulsar brokers default to a limit of 5 MB batch size. EMQX's default value is less than 5 MB in order to compensate Pulsar message encoding overheads (especially when each individual message is very small). When a single message is over the limit, it is still sent (as a single element batch).",
```

Comment:
```suggestion
      "desc": "Specify the limit on the number of bytes collected in a batch. EMQX has set the default value to less than 5 MB to account for encoding overheads when handling small messages. If a single message exceeds the batch size limit, it will still be sent as a single-element batch, indicating it will be treated as a separate batch containing only that specific message.",
```